components/clarity-repl/src/repl/interpreter.rs

```diff
@@ -282,31 +282,20 @@ impl ClarityInterpreter {
                 // If they don't match, report a warning and use v1. Once v2 is battle-tested,
                 // this can be removed. If there are errors when parsing, then the ASTs are not
                 // expected to match, and v2 is still used.
-                let (ast, diagnostics, success) =
-                    ast::build_ast(&contract_identifier, &snippet, &mut ());
-                let (ast_old, mut diagnostics_old, success_old) =
-                    match clarity::ast::build_ast(&contract_identifier, &snippet, &mut ()) {
-                        Ok(res) => (res, vec![], true),
-                        Err(error) => (
-                            ContractAST::new(contract_identifier, vec![]),
-                            vec![error.diagnostic],
-                            false,
-                        ),
-                    };
-                if (success && !ast.equivalent(&ast_old)) || success != success_old {
-                    diagnostics_old.push(Diagnostic {
-                        level: Level::Warning,
-                        message: r#"conflict between parser versions, reverting to parser v1
-Help improve clarinet by sharing the code that triggered this warning:
-https://github.com/hirosystems/clarinet/issues/new/choose"#
-                            .to_string(),
-                        spans: vec![],
-                        suggestion: None,
-                    });
-                    (ast_old, diagnostics_old, success_old)
-                } else {
-                    (ast, diagnostics, success)
-                }
+                // let (ast, diagnostics, success) =
+                //     ast::build_ast(&contract_identifier, &snippet, &mut ());
+                // let (ast_old, mut diagnostics_old, success_old) =
+                //     match clarity::ast::build_ast(&contract_identifier, &snippet, &mut ()) {
+                //         Ok(res) => (res, vec![], true),
+                //         Err(error) => (
+                //             ContractAST::new(contract_identifier, vec![]),
+                //             vec![error.diagnostic],
+                //             false,
+                //         ),
+                //     };
+
+                // switch to v2 for now for performances
+                ast::build_ast(&contract_identifier, &snippet, &mut ())
```

Comment:
Using exclusively v2 for now since it speeds up the testing process

Reply:
I don't think we should merge this to develop though, right?

Reply:
Yep sure, I'll remove it when it's time to merge. Or maybe add a condition to have this behavior on wasm? At least during the "alpha" phase

Reply:
On second thought, maybe it is fine to just delete this comparison with v1. We are confident enough in v2 that it will be merged upstream, so it should be fine to remove this check now.

Reply:
Wait, no but there are still some things that v2 allows but will fail in v1... we need clarinet to catch those.

Reply:
Should we take an opinionated stance in this specific tool, and just rely on `v2`? In my mind pushing for best practices is fine in the context of code edition, no?

Reply:
The problem though is that in the PR process to stacks-blockchain, the v2 parser got some changes that are not backwards compatible, but that was okay since it is gated for 2.1. If we start using it before 2.1, then it would allow some things that would be rejected by the node. For example, some spacing rules are different:

```clarity
{foo:1}
```

This is valid with parser v2, but not with v1. If I get the merge finished, then we can just use the merged version which will use only v2 for 2.1 epoch and only v1 for before.

Reply:
I ended up restoring the initial logic, but we skip the v1 check in the wasm context